core/trino-main/src/test/java/io/trino/sql/query/QueryAssertions.java

```diff
@@ -385,9 +385,14 @@ private static void assertTypes(MaterializedResult actual, List<Type> expectedTy
         }
 
         public QueryAssert returnsEmptyResult()
+        {
+            return returnsRowsCount(0);
+        }
+
+        public QueryAssert returnsRowsCount(int expected)
```

Comment:
i think this method is not necessary -- at least not in this PR, since the only two usages should be better served with returnsEmptyResult and exact result check respectively.